azresources/containers/aks/aks-without-cmk.bicep

```diff
@@ -79,7 +85,19 @@ param containerInsightsLogAnalyticsResourceId string = ''
 @description('Enable encryption at host (double encryption).  Default: true')
 param enableEncryptionAtHost bool = true
 
-resource akskubenet 'Microsoft.ContainerService/managedClusters@2021-07-01' = {
+var podCidra = contains(networkPlugin, 'azure') ? null : podCidr
```

Comment:
What does "a" in `podCidra` mean? Can you write out the variable in full so it's self describing?